.env.example

```diff
@@ -0,0 +1 @@
+SDK_TOKEN_FACTORY_SECRET=
```

Comment:
If we decide to pull translations from Lokalise at runtime we could later on add `LOKALISE_TOKEN` and `LOKILISE_PROJECT_ID` here